CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 6.2.3 (2022-8-18)
```

Comment:
Wait to merge until we run the release process.

Reply:
👍